src/script/builder/tir/prim_func_frame.h

```diff
@@ -32,13 +32,17 @@ class PrimFuncFrameNode : public TIRFrameNode {
   Array<tvm::tir::Var> args;
   Type ret_type;
   Map<tvm::tir::Var, tvm::tir::Buffer> buffer_map;
+  Map<tvm::tir::Var, tvm::tir::Buffer> preflattened_buffer_map;
+  DictAttrs attrs;
```

Comment:
Let's use Map here and later converts it to DictAttrs :-)

```suggestion
  Map<String, ObjectRef> attrs;
```